python/bootstrap.py

```diff
@@ -121,10 +120,6 @@ def setup_projects(backend):
     exit(1)
   setup_find_fields(backend, lucidfind_collection_id)
 
-  status = backend.create_collection(threads_collection_id)
-  if status == False:
-    exit(1)
-  setup_thread_fields(backend, threads_collection_id)
```

Comment:
@xiaoyehhuang we should also remove the method definition at the top of the file (around line 41, I think)